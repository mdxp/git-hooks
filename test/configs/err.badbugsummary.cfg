[core]
	repositoryformatversion = 0
	filemode = true
	bare = false
	logallrefupdates = true
	ignorecase = true
[gate]
	name = gate
	notify = invalidemailaddress@abadaddress
	user-check = skip
[test]
	comment = 1234 invalid bug summary
