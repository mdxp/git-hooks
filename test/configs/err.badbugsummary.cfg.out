remote: Bug summaries must match with data in Bugzilla[K
remote: - 1234 invalid bug summary[K
remote: + 1234 Top Search  on Logo/Menu bar - "Search" Application[K
remote: 
remote: Use the following command to edit the last commit message:[K
remote:   git commit --amend --reset-author[K
remote: 
